x-pack/plugins/observability_solution/slo/docs/openapi/slo/bundled.json

```diff
@@ -1740,11 +1740,17 @@
           "target": {
             "description": "the target objective between 0 and 1 excluded",
             "type": "number",
+            "minimum": 0,
+            "maximum": 100,
+            "exclusiveMinimum": true,
+            "exclusiveMaximum": true,
```

Comment:
I don't know who controls the definition for this but "exclusive" indicates to me that the minimum/maximum numbers are _not_ included in the range (i.e. excluded). Is `exclusiveMinimum` / `exclusiveMaximum` correct?